src/data/events/archive/winter2021/hoth-8.yml

```diff
@@ -0,0 +1,87 @@
+- name: Hack off the Hill 8 Workshops
+  quarter: Winter 2021
+  mainLink: https://www.youtube.com/watch?v=ryXcoMyHfA8&list=PLPO7_kXilXFZMFFnXUWAAQwt0Bd4UDAaF
+  tags: ['hackathon']
+  director:
+  - Asha Kar
+
+  workshops:
+  - name: Getting Started with a Hackathon
+    youtube: https://youtu.be/ryXcoMyHfA8
```

Comment:
Can we also add READMEs and slides? E.g., https://github.com/uclaacm/hoth8-intro-to-html-css-js and https://docs.google.com/presentation/d/1Y9rvAXjyeZOADBceqlAr9eeakO7URGWwu4QRulgvhSs/edit are for Jakob's web dev workshops.